### PR TITLE
Add latest backend errors to cowswap

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Adds latest errors found in newest API spec from here: https://github.com/gnosis/gp-v2-services/blob/main/crates/orderbook/openapi.yml#L740

Screenshot:
image

### TEST
1. XDAI chain
2. select 0xMR as in and XGTv2 as out
3. trade
4. should see insufficient liquidty error as opposed to Error try again later error

### CODE REVIEW

**Review by MareenG — APPROVED** [review]
Works nicely.

**Review by alfetopito — APPROVED** [review]
Could you also do the one for buying ETH and sending it to a Smart Contract?

**Comment by elena-zh** [comment]
> Could you also do the one for buying ETH and sending it to a Smart Contract?

To add: issue number #1612

**Comment by W3stside** [comment]
@alfetopito e478c22

File: src/custom/api/gnosisProtocol/errors/OperatorError.ts [hunk 1]
```diff
@@ -6,21 +6,27 @@ export interface ApiErrorObject {
 }
 
 // Conforms to backend API
-// https://github.com/gnosis/gp-v2-services/blob/0bd5f7743bebaa5acd3be13e35ede2326a096f14/orderbook/openapi.yml#L562
+// https://github.com/gnosis/gp-v2-services/blob/d932e11c9a2125fdba239530be7684799f694909/crates/orderbook/openapi.yml#L801
+// and
+// https://github.com/gnosis/gp-v2-services/blob/d932e11c9a2125fdba239530be7684799f694909/crates/orderbook/openapi.yml#L740
```
Comment (anxolin) [inline src/custom/api/gnosisProtocol/errors/OperatorError.ts:11]: Nice you add the blob. As a nitpick, I would also make a comment with something: See latest version and a link to their main branch file where they always have the latest

This way we can easily check if there's sth new

**Review by anxolin — APPROVED** [review]

**Comment by elena-zh** [comment]
Seems that it works: at least, the explanation is much better than it was before. 
image

**Comment by anxolin** [comment]
@W3stside should we include this one in 1.5 too?

**Comment by alfetopito** [comment]
> @alfetopito [e478c22](https://github.com/gnosis/cowswap/commit/e478c22dbd59be8593b1caffd3976c3db6649f91)
> 
> no clue how to test it works tho, please let me know if it's ok

Elena tested it already, but it's easy: Try to place and order with Gnosis Safe or add as recipient a SmartContract address

**Comment by anxolin** [comment]
@W3stside i just rebased your branch to 1.5 and merged, i hope is alright

**Comment by W3stside** [comment]
@anxolin all good was going to do the same, thanks

**Pull request merged**